app/views/govuk_publishing_components/components/layout_header/_navigation_items.html.erb

```diff
@@ -1,17 +1,24 @@
 <% if navigation_items.any? %>
-  <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
-  <%= tag.nav(class: "gem-c-header__nav", 'aria-label': navigation_aria_label ? navigation_aria_label : nil ) do %>
+  <div class="govuk-header__navigation-container">
```

Comment:
What's the need for this element? I assume the styles are coming from `govuk-frontend` - was this an omission that you're correcting?

Reply:
Good shout. No need for this element! I originally thought it was necessary to add an extra container around the `nav` in order to achieve what I needed but I can remove it now.